ksiren/src/main/java/com/brightspace/ksiren/Field.kt

```diff
@@ -66,6 +75,18 @@ data class Field(
 			return obj
 		}
 	}
+}
 
-	override fun toJson() = JsonUtils.toJson(this)
+internal data class MutableField(
+	override val name: String,
+	override var value: String?,
```

Comment:
I suppose `Field` itself could have `value` converted to a `var`, but I'd prefer to keep the exported/original classes read-only